src/components/PetCardComponent.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="pet-selection">
+    <div class="pet-card" @click="selectPet">
+      <img :src="pet.image" :alt="pet.name">
+      <div class="pet-card-name">
+        <p>{{ pet.name }}</p>
+        <img src="../assets/icon_audio.svg" alt="Audio">
+      </div>
+    </div>
+  </div>
+</template>
+
+
+<script setup lang="ts">
+import { defineProps, defineEmits } from 'vue';
+
+const props = defineProps<{
+  pet: { name: string; image: string };
+}>();
+
+const emit = defineEmits(['pet-selected']);
+
+const selectPet = () => {
+  emit('pet-selected', props.pet); 
+};
+</script>
+
+
+<style scoped>
+.pet-selection {
+  margin-top: 20px;
+  display: flex;
+  justify-content: space-around;
+
+  .pet-card {
```

Comment:
![image](https://github.com/schule4-0/pets/assets/131030998/1d568590-48f8-4b51-976f-e564ac23a572)
The cards should align on the left with the header and subtext